src/main/dashboardServer/proxyUtils.ts

```diff
@@ -0,0 +1,61 @@
+import { Options } from 'http-proxy-middleware';
+
+import Logging from '@/utils/logging';
+
+const console = Logging.dashboardServer;
+
+export const proxyOpts = (target: string): Options => {
+  return {
+    target,
+    followRedirects: true,
+    secure:          false,
+    logLevel:        'debug',
+    onProxyReq,
+    onProxyReqWs,
+    onError,
+  };
+};
+
+export const proxyWsOpts = (target: string): Options => {
+  return {
+    ...proxyOpts(target),
+    ws:           false,
+    changeOrigin: true,
+  };
+};
+
+export const proxyMetaOpts = (target: string): Options => {
```

Comment:
How is this different from `proxyOpts`? (If you want the better name, just do `export const proxyMetaOpts = proxyOpts`?)

Reply:
I think this was originally intended to be a bit different, but ended up being exactly the same after pruning. I'll keep `proxyMetaOpts` around just for the name..